src/workerd/server/workerd.c++

```diff
@@ -754,6 +754,10 @@ class CliMain final: public SchemaFileImpl::ErrorReporter {
         "<const-name>", CLI_METHOD(setConstName));
   }
 
+  bool shouldEnableAllAutogates() {
```

Comment:
As is I don't see how this would work for KJ_TEST as Kenton requested.

Reply:
> Should we consider using an environment variable instead, so that we can also run KJ tests with all-autogates?

The idea of moving to env var was specifically so that it would be useful for KJ_TESTs